.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,40 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
-1. If this is your first time, read our contributor guidelines https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
-2. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
-3. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
-4. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+<!--
+
+Thanks for opening a pull request! Here are some tips to get everything merged smoothly:
+
+1. Read our contributor guidelines: https://cert-manager.io/docs/contributing/
+
+2. Make sure your commits are signed off: https://cert-manager.io/docs/contributing/sign-off/
+
+3. If the PR is unfinished, raise it as a draft or prefix the title with "WIP:" so it's clear to everyone.
+
+4. Be sure to allow edits from maintainers so it's easier for us to help: https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork
+
 -->
 
-**What this PR does / why we need it**:
+### Pull Request Motivation
+
+<!-- Explain the motivation behind this PR. If there's a related issue or PR, link to it here! -->
 
-**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
+### Kind
 
-**Special notes for your reviewer**:
+<!-- Pick a kind which best describes your PR so it can be labelled correctly, and delete the others: -->
+
+/kind cleanup
+/kind bug
+/kind feature
+/kind documentation
```

Comment:
```suggestion
<!-- During a release, we group merged PRs by category in the release notes.

Pick a kind which best describes your PR so it can be labelled correctly, and delete the others. If you are unsure, delete them all and a maintainer will add one. -->

/kind cleanup
/kind bug
/kind feature
/kind documentation
/kind design
/kind flake
```

Suggestion: Maybe this makes it more clear?

Reply:
Although if prow does this, perhaps it's just best to leave it out entirely!

![image](https://user-images.githubusercontent.com/333465/150826993-e3528a3d-b45b-4c41-bf55-4cb5d8b7b08b.png)